products/views.py

```diff
@@ -1 +1,246 @@
+import json
+
+from django.views   import View
+from django.http    import JsonResponse, HttpResponse
+
+from products.models import (
+                                Product,
+                                Skin,
+                                Feel,
+                                Ingredient,
+                                Aroma,
+                                SubCategory,
+                                Category,
+                                Media,
+                                Texture,
+                                
+)
+                            
+
+class ProductListView(View):
+    def get(self, request):
+        try:
+            products = Product.objects.all()
+
+            my_product = []
+            for product in products:
+
+                subcategories    = product.subcategories.all()
+                subcategory_info = {f"subcategory_{i}": subcat.name for i, subcat in enumerate(subcategories)}
+                
+                if subcategories:
+                    category = subcategories[0].category.name
+
+                skin_types = product.skin_types.all()
+                skin_info  = {f"type_{i}": skin.name for i, skin in enumerate(skin_types)}
+
+                feels     = product.feels.all()
+                feel_info = {f"feel_{i}": feel.name for i, feel in enumerate(feels)}
+                
+                ingredients     = product.ingredients.all()
+                ingredient_info = {f"ingredient_{i}": ingredient.name for i, ingredient in enumerate(ingredients)}
```

Comment:
```suggestion
                skin_info  = {f"type_{i}": skin.name for i, skin in enumerate(product.skin_types.all())}
                feel_info = {f"feel_{i}": feel.name for i, feel in enumerate(product.feels.all())}
                ingredient_info = {f"ingredient_{i}": ingredient.name for i, ingredient in enumerate(product.ingredients.all())}
```
`skin_types`, `feels`, `ingredients`와 같은 변수들이 여러번 사용되는 것이 아니라면 불필요하게 변수의 개수를 늘리는 것보다는 한 번에 합치시는 방법을 추천드립니다. 아래 다른 변수들에도 적용 추천드립니다 :)